schema/Retail/RET11/on_init.ts

```diff
@@ -538,24 +538,146 @@ export const FnBonInitSchema = {
+                                pattern: '^[0-9]+(\\.[0-9]+)?$',
+                                errorMessage: 'Value for max_liability_cap must be a number',
+                              },
+                            },
+                          },
                         },
-                      },
-                      required: ['code', 'value'],
+                        {
+                          if: {
+                            properties: {
+                              code: {
+                                const: 'mandatory_arbitration',
+                              },
+                            },
+                          },
+                          then: {
+                            properties: {
+                              value: {
+                                type: 'string',
+                                pattern: '^(true|false)$',
+                                errorMessage: "Value for mandatory_arbitration must be either 'true' or 'false'",
+                              },
+                            },
+                          },
+                        },
+                        {
+                          if: {
+                            properties: {
+                              code: {
+                                const: 'court_jurisdiction',
+                              },
+                            },
+                          },
+                          then: {
+                            properties: {
+                              value: {
+                                type: 'string',
+                                pattern: '^[A-Za-z\\s]+$',
+                                errorMessage: 'Value for court_jurisdiction must be alphabetic characters only',
+                              },
+                            },
+                          },
+                        },
+                        {
+                          if: {
+                            properties: {
+                              code: {
+                                const: 'delay_interest',
+                              },
+                            },
+                          },
+                          then: {
+                            properties: {
+                              value: {
+                                type: 'string',
+                                pattern: '^[0-9]+(\\.[0-9]+)?$',
+                                errorMessage: 'Value for delay_interest must be a number',
+                              },
+                            },
+                          },
+                        },
+                        {
+                          if: {
+                            properties: {
+                              code: {
+                                const: 'tax_number',
+                              },
+                            },
+                          },
+                          then: {
```

Comment:
only tax_number and provider_tax_number are required right now in bpp_terms excluding rest of the tnCs